### ISSUE TITLE
 Inappropriate absence of fever

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6121]
**Preferred term label:**
 Inappropriate absence of fever

 -- see https://pubmed.ncbi.nlm.nih.gov/31235509/

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**
https://hpo.jax.org/app/browse/term/HP:0004370

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
